log-manager/src/main/java/io/airlift/log/StaticFormatter.java

```diff
@@ -66,6 +81,13 @@ public String format(LogRecord record)
                 .append('\t')
                 .append(record.getMessage());
 
+        if (hasLogAnnotations()) {
+            stringWriter.append('\t')
```

Comment:
The annotations should probably go before the message, which may contain new lines in some cases.

Reply:
My only hesitation with that is if a parser is splitting on tabs, it would mean the ordering of fields would change if the annotations are present or not.